src/Symfony/Bridge/PhpUnit/CoverageListener.php

```diff
@@ -29,7 +29,7 @@ class CoverageListener implements TestListener
     public function __construct(?callable $sutFqcnResolver = null, bool $warningOnSutNotFound = false)
     {
         $this->sutFqcnResolver = $sutFqcnResolver ?? static function (Test $test): ?string {
-            $class = \get_class($test);
+            $class = $test::class;
```

Comment:
We can't apply this fixer to the PHPUnit bridge. We still need to support PHPUnit 7 here.

Reply:
ack, any plans to drop support for legacy versions ?

Reply:
Plans, yes. 🙂